thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/FetchAnomaliesInRangeAndOutputCSV.java

```diff
@@ -0,0 +1,193 @@
+    sb.append("[");
+    if(!dm.isEmpty()) {
+      for (Map.Entry<String, String> entry : dm.entrySet()) {
+        sb.append(entry.getKey() + ":");
+        sb.append(entry.getValue() + "|");
+      }
+      sb.deleteCharAt(sb.length() - 1);
+    }
+    sb.append("]");
+    return sb.toString();
+  }
+
+  public static void init(File persistenceFile) throws Exception {
+    DaoProviderUtil.init(persistenceFile);
+    anomalyFunctionDAO = DaoProviderUtil
+        .getInstance(com.linkedin.thirdeye.datalayer.bao.jdbc.AnomalyFunctionManagerImpl.class);
+    rawAnomalyResultDAO = DaoProviderUtil
+        .getInstance(com.linkedin.thirdeye.datalayer.bao.jdbc.RawAnomalyResultManagerImpl.class);
+    mergedAnomalyResultDAO = DaoProviderUtil
+        .getInstance(com.linkedin.thirdeye.datalayer.bao.jdbc.MergedAnomalyResultManagerImpl.class);
+  }
+
+  public static void outputResultNodesToFile(File outputFile, List<FetchMetricDataAndExistingAnomaliesTool.ResultNode> resultNodes)
+    throws IOException{
+    BufferedWriter bw = new BufferedWriter(new FileWriter(outputFile));
+
+    bw.write(String.join(",", resultNodes.get(0).getSchema()));
+    bw.newLine();
+    for (FetchMetricDataAndExistingAnomaliesTool.ResultNode n : resultNodes){
+      bw.write(n.toString());
+      bw.newLine();
+    }
+    bw.close();
+  }
+  /**
+   * Ouput merged anomaly results for given metric and time range
+   * @param args List of arguments
+   *             0: path to persistence file
+   *             1: collection name
+   *             2: metric name
+   *             3: start time in ISO format
+   *             4: end time in ISO format
+   *             5: (optional): Output path
+   */
+  public static void main(String args[]){
+    if(args.length < 5){
+      System.out.println("Insufficient number of arguments");
+      return;
+    }
+
+    FetchMetricDataAndExistingAnomaliesTool thirdEyeDAO = null;
+    try {
+      thirdEyeDAO = new FetchMetricDataAndExistingAnomaliesTool(new File(args[0]));
+    }
+    catch (Exception e){
+      System.out.println(e.getMessage());
+      return;
+    }
+
+    String collection = args[1];
+    String metric = args[2];
+    String output_folder = DEFAULT_OUTPUT_FOLDER;
+    DateTime startTime = ISODateTimeFormat.dateTimeParser().parseDateTime(args[3]);
+    DateTime endTime = ISODateTimeFormat.dateTimeParser().parseDateTime(args[4]);
+    DateTimeFormatter fmt = DateTimeFormat.forPattern("yyyy-MM-dd");
+    if(args.length >= 6 && (new File(args[5])).exists()){
+      output_folder = args[5];
+    }
+
+
+
+    // Print Merged Results
+    List<FetchMetricDataAndExistingAnomaliesTool.ResultNode> resultNodes = thirdEyeDAO.fetchMergedAnomaliesInRange(
+        collection, metric, args[3], args[4]);
+
+    System.out.println("Printing merged anaomaly results from db...");
+    String outputname = output_folder +
+        "merged_" + args[1] + "_" + fmt.print(startTime) + "_" + fmt.print(endTime) + ".csv";
```

Comment:
Use "collection" which has been assigned value of args[1]. Assign values of args[3] and[4] to variables, as they are being used in multiple places, and helps in giving clarity as to what is being passed as params